pkg/relayer/config/relayminer_configs_reader.go

```diff
@@ -33,35 +31,47 @@ type RelayMinerConfig struct {
 func ParseRelayMinerConfigs(configContent []byte) (*RelayMinerConfig, error) {
 	var yamlRelayMinerConfig YAMLRelayMinerConfig
 
+	if len(configContent) == 0 {
```

Comment:
NIT: While I (totally subjectively) prefer this approach better (i.e. over `configConftent == ""`), there seems to be several similar kinds of comparisons made throughout this function. No strong opinion either way but regardless, I think we should be consistent (at least at this scale).

Reply:
+1 for consistency, w/e approach we take